Esther125/CloudDrop
# Feature/jlin/cns 54 staging area page

## Change
- Add staging area page
  - FileList
  - Scroll down and more file
  - no  more file, stop call API
  - Download Button
  - no file => 暫無檔案

## Test
1. start backend and frontend
2. click Temporary Storage
3. 
- if user doesn't have file in s3
![image](https://github.com/user-attachments/assets/b21229c0-cde0-4393-a1e8-60346da07ac9)
- if user have many file in s3
(Comment out line 9 and uncomment line 10.)
```
// const userID = store.user.id;
const userID = '12346';
```
<img width="1169" alt="image" src="https://github.com/user-attachments/assets/70054b4d-5f42-4ce7-b67c-c7a84a7f1428" />
and Scroll down, you will see more file
<img width="1163" alt="image" src="https://github.com/user-attachments/assets/730816c8-8711-4bcb-8ee3-455225a106d4" />
4. click 下載 button, then download the file
<img width="1165" alt="image" src="https://github.com/user-attachments/assets/a93891c0-5c6c-4f3d-8980-6743d991d6ca" />
